data-catalog-backend-app/src/main/java/no/nav/data/catalog/backend/app/codelist/CodelistService.java

```diff
@@ -40,18 +41,33 @@ public void refreshCache() {
 		});
 	}
 
-	public Codelist save(CodelistRequest request) {
-		codelists.get(request.getList()).put(request.getCode(), request.getDescription());
-		return repository.save(request.convert());
+	public List<Codelist> save(List<CodelistRequest> requests) {
+		requests.forEach(request -> codelists.get(request.getList())
+				.put(request.getCode().toUpperCase(), request.getDescription()));
+		return repository.saveAll(requests.stream()
+				.map(CodelistRequest::convert)
+				.collect(Collectors.toList()));
 	}
 
-	public Codelist update(CodelistRequest request) {
-		codelists.get(request.getList()).put(request.getCode(), request.getDescription());
+	public List<Codelist> update(List<CodelistRequest> requests) {
```

Comment:
Add trim and check for null values

Reply:
Added trim :) 
Null values will be validated before the request get here. 
Validation is commented out until branch #42